coremltools/converters/tensorflow/test/test_tf_2x.py

```diff
@@ -107,6 +108,108 @@ def __call__(self, x):
                     return x * 3.
         self._test_coreml(model())
 
+    def test_einsum_transpose(self):
```

Comment:
Since there are many einsum tests (which is great!) do you think it would be better to have a separate class for einsum to group all these tests?

Reply:
good!